### PR TITLE
refactor: mv item record parsing to item page

### PR DESCRIPTION
**Pull request by rwd**
* move Record API response parsing from "plugin" to item page
  * as that's the only place it's used, and
  * it leaves the Record API "plugin" leaner and more reusable as an API client
* simplify the UI/metadata language handling for translation on the item page
* simplify the LangMap language code selection to not need to be aware of the UI language, instead having Vue components use a new mixin method `langAttribute`  to set HTML `lang` attributes to `null` when the same as the i18n locale
* display provider metadata first in the metadata table, moving enrichment entity links to the end of each row

### CODE REVIEW

File: packages/portal/src/pages/item/_.vue [hunk 1]
```diff
@@ -356,27 +357,207 @@
...
+            return this.$error(error, { scope: 'item' });
+          }
+        }
+        const edm = data.object;
+
+        if (this.identifier !== edm.about) {
+          return this.redirectToAltRoute({ params: { pathMatch: edm.about?.slice(1) } });
+        }
+
+        this.identifier = edm.about;
```
Comment (LeoniePeters) [inline packages/portal/src/pages/item/_.vue:384]: Why is this needed? Just before this line there is a redirect in place for when the `identifier` and `edm.about` are different. Wouldn't that already resolve them to be the same?

File: packages/portal/src/pages/item/_.vue [hunk 2]
```diff
@@ -356,27 +357,207 @@
...
+          .map((entity) => reduceLangMapsForLocale(entity, this.metadataLanguage));
+      },
+
+      extractEuropeanaProxy(edm) {
+        const europeanaProxy = this.findProxy(edm.proxies, 'europeana') || {};
+
+        // Europeana proxy only really needed for the translate profile
+        if (this.translatingMetadata) {
+          for (const field in europeanaProxy) {
+            if (europeanaProxy[field][this.metadataLanguage]) {
```
Comment (LeoniePeters) [inline packages/portal/src/pages/item/_.vue:433]: Should this be `translationLanguage` instead of `metadataLanguage`?

File: packages/portal/src/pages/item/_.vue [hunk 3]
```diff
@@ -356,27 +357,207 @@
       },
 
       async fetchMetadata() {
+        const params = {};
+        if (this.translatingMetadata) {
+          params.profile = 'translate';
+          params.lang = this.metadataLanguage;
```
[the remark this answers is not in the document]
Comment (rwd) [reply packages/portal/src/pages/item/_.vue:363]: They will be the same value if translatingMetadata is true, so either will work here.

File: packages/portal/src/pages/item/_.vue (hunk 1 above)
Comment (rwd) [reply packages/portal/src/pages/item/_.vue]: Indeed. That's removed.

File: packages/portal/src/pages/item/_.vue (hunk 2 above)
Comment (rwd) [reply packages/portal/src/pages/item/_.vue:432]: They will be the same value if translatingMetadata is true, so either will work here.

File: packages/portal/src/pages/item/_.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (LeoniePeters) [reply packages/portal/src/pages/item/_.vue:363]: Yes, they both work. I'm just thinking that conceptually it would be more correct to use `translationLanguage`. `translationLanguage` is defined by the lang param only, whereas `metadataLanguage` has the UI locale as fallback, which should never be passed as lang param (right?). At the moment it never will, due to the current logic. I think it would help though, to use the specific language prop to keep the difference between `translationLanguage` and `metadataLanguage` clear. I had to untangle that difference a little while reviewing and would like to limit any future confusion.

[the remark this answers is not in the document]
Comment (LeoniePeters) [reply packages/portal/src/pages/item/_.vue:363]: (That also applies for my other similar comment)

[the remark this answers is not in the document]
Comment (rwd) [reply packages/portal/src/pages/item/_.vue]: Agreed, and made it so.

**Review by lbiedinger — APPROVED** [review]
